test/utils.py

```diff
@@ -1153,3 +1153,9 @@ def prepare_spec_arguments(spec, arguments, opname, entity_map, with_txn_callbac
                 raise AssertionError(f"Unsupported cursorType: {cursor_type}")
         else:
             arguments[c2s] = arguments.pop(arg_name)
+
+
+def set_fail_point(client, command_args):
```

Comment:
Can you use the [with self.fail_point()](https://github.com/mongodb/mongo-python-driver/blob/7936708d97b6bb77a0d6fe6c378f81594c34a7c5/test/__init__.py#L988) helper instead?

Reply:
We need to set the fail point on each individual mongos to ensure the failure occurs on every node. `with self.fail_point()` does not set the fail point correctly.